aip/client-libraries/4222.md

```diff
@@ -154,6 +160,46 @@ If none of the routing parameters matched their respective fields, the routing h
 Much like URL parameters, if there is more than one key-value pair to be sent, the `&`
 character is used as the separator.
 
+### `path_template` syntax
+
+As seen in the above examples, the `path_template` can use a variety of symbols
+that are interpreted by code generators during conversion to regular expressions
+or non-regular expression matcher implementations. The syntax for these symbols
+is as follows:
+
+- The only acceptable segment delimiter is `/`.
+  - The last symbol in a `path_template` **may** be a delimiter - it will be 
+    ignored.
+  - A segment **must not** represent a complex resource ID as described in
+    [AIP-4231][]. A Generator **should** emit an error in this case.
```

Comment:
shift left since this should be not under "segment delimeter"

Reply:
Good catch, I moved it to the bottom (after definition of "segment") and shifted left.